views/tables/ChartsTable/TableMenu.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { Button, Menu, MenuItem, MenuList, Typography } from '@mui/material'
+import ContentCopyIcon from '@mui/icons-material/ContentCopy'
+import MoreVertIcon from '@mui/icons-material/MoreVert'
+import EditIcon from '@mui/icons-material/Edit'
+import DeleteIcon from '@mui/icons-material/Delete'
+import ShareIcon from '@mui/icons-material/Share'
+import { Link } from 'react-router-dom'
+
+import { routes } from '../../routes/routes'
+import ChartModel from '../../models/ChartModel'
+
+const MENU_ITEM_STYLES = { display: 'flex', gap: '32px' }
```

Comment:
should we put these in constants..?

Reply:
I don't think so, this is a constant for this component only.